DataConnectors/Gitlab/README.md

```diff
@@ -0,0 +1,19 @@
+## Description
+This log source uses logic app workflow with log analytics send data plugin. This Gitlab connector uses 3 types of audit logs (Project Audit, Group Audit and user events). For additional audit logs, you can modify the workflow with additional apis. For more information on Gitlab audit events, refer to https://docs.gitlab.com/ee/api/audit_events.html
```

Comment:
We should stylize their name as they do.

https://about.gitlab.com/handbook/marketing/corporate-marketing/brand-activation/trademark-guidelines/#also-you-must-comply-with-these-additional-guidelines

> ### Also, you must comply with these additional guidelines:
> ...
> - Never hyphenate, make a possessive of, pluralize, abbreviate, or change the capitalization of our Trademarks, or use our Trademarks as verbs. For example, do not refer to our platform as “gitlab”.


They stylize their name as: `GitLab`

We should change the directory name to match, and change all instances of `Gitlab` to `GitLab` in the README.md as well as throughout the `azuredeploy.json` (except in the case of a URL, which should be all lowercase).

```suggestion
This log source uses logic app workflow with log analytics send data plugin. This GitLab connector uses 3 types of audit logs (Project Audit, Group Audit and user events). For additional audit logs, you can modify the workflow with additional APIs. For more information on GitLab audit events, refer to https://docs.gitlab.com/ee/api/audit_events.html
```